### PR TITLE
fixing some coala errors

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

**Comment by goern** [comment]
@CermakM there is a lot of coala errors... pls have a look https://zuul.thoth-station.ninja/logs/18/18/1c6038aa761c2a3d42476afb4d9813427c2d4fd2/check/thoth-coala/c495f60/ara-report/result/ec7f44cb-5f49-4f23-a336-a0065e164239/

**Comment by CermakM** [comment]
Thanks Christoph, will do!

File: openshift/buildConfig-template.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ metadata:
     description: >
       This is Osiris API BuildConfig Template.
     openshift.io/display-name: 'Thoth: Osiris API - BuildConfig'
+    version: 0.1.0
```
Comment (CermakM) [inline openshift/buildConfig-template.yaml:9]: What this version corresponds to? Is it a version of the template itself? (same for ConfigMap template)

Comment (goern) [reply openshift/buildConfig-template.yaml:9]: ja

Comment (CermakM) [reply openshift/buildConfig-template.yaml:9]: Cool, thanks!

**Pull request merged**